omnibus/.kitchen.yml

```diff
@@ -2,35 +2,89 @@ driver:
+    cpus: 4
+    memory: 4096
   synced_folders:
     - ['..', '/home/vagrant/inspec']
+    - ['../../omnibus', '/home/vagrant/omnibus']
+    - ['../../omnibus-software', '/home/vagrant/omnibus-software']
 
 provisioner:
   name: chef_zero
   require_chef_omnibus: true
+  attributes:
+    vagrant:
+      this_key_exists_so_we_have_a_vagrant_key: true
+    omnibus:
+      build_user:          vagrant
+      build_user_group:    vagrant
+      build_user_password: vagrant
+      install_dir:         /opt/inspec
 
 platforms:
-  - name: centos-7.1
+  - name: centos-7.2
     run_list: yum-epel::default
-  - name: centos-6.6
+  - name: centos-6.7
     run_list: yum-epel::default
-  - name: debian-7.8
+  - name: debian-8.2
     run_list: apt::default
-  - name: debian-6.0.10
+  - name: debian-7.9
     run_list: apt::default
   - name: ubuntu-14.04
     run_list: apt::default
   - name: ubuntu-12.04
     run_list: apt::default
+  # The following (private) boxes are shared via Atlas and are only
+  # available to users working for Chef. Sorry, it's about software licensing.
+  #
+  # Chef-internal users, you will need to:
+  # 1.  Create an Atlas account:  https://atlas.hashicorp.com/
+  # 2.  Ping #eng-services-support with your Atlas account name
+  #     to be added to the relevant team in Atlas,
+  # 3.  Do `vagrant login` with your Atlas creds so that you can download
+  #     the private boxes.
+  #
+  # The Mac OS X boxes are VMware only also. You can enable VMware Fusion
+  # by activating the `.kitchen.vmware.yml` file with the `KITCHEN_LOCAL_YAML`
+  # environment variable:
+  #
+  #   KITCHEN_LOCAL_YAML=.kitchen.vmware.yml kitchen converge inspec-macosx-109
+  #
+  <% %w(
+    10.9
+    10.10
+    10.11
+  ).each do |mac_version| %>
+  - name: macosx-<%= mac_version %>
+    driver:
+      box: chef/macosx-<%= mac_version %> # private
+      synced_folders:
+      - ['..', '/Users/vagrant/chef']
+      - ['../../omnibus', '/Users/vagrant/omnibus']
+      - ['../../omnibus-software', '/Users/vagrant/omnibus-software']
+  <% end %>
+
+  # By adding an `i386` to the name the Omnibus cookbook's `load-omnibus-toolchain.bat`
+  # will load the 32-bit version of the MinGW toolchain.
+  <% [ '', '-i386' ].each do |win_suffix| %>
+  - name: windows-2012r2-standard<%= win_suffix %>
+    driver:
+      box: chef/windows-server-2012r2-standard # private
+      synced_folders:
+      # We have to mount this repos enclosing folder as the Omnibus build
+      # gets cranky if the mounted source folder is a symlink. This
+      # mounts at `C:\vagrant\code` and the inspec source folder is available
+      # at `C:\vagrant\code\inspec`
+      - ['../..', '/vagrant/code']
+    provisioner:
+      attributes:
+        omnibus:
+          build_user:          vagrant
+          build_user_group:    Administrators
+          build_user_password: vagrant
+          install_dir:         /opscode/inspec
+  <% end %>
 
 suites:
-  - name: default
+  - name: inspec
```

Comment:
None in particular - just cosmetic.  If I've a bunch of VMs running, setting the suite name correctly helps identify the VM inside virtual box.  Otherwise everything looks like "kitchen-omnibus-default-windows-2012r2-standard..."